README.md

```diff
@@ -32,9 +32,9 @@ Differing from the current official [Nix](https://github.com/NixOS/nix) installe
 * In `nix.conf`:
   + the `auto-allocate-uids`, `nix-command` and `flakes` features are enabled
   + `bash-prompt-prefix` is set
-  + `auto-optimise-store` is set to `true`
+  + `auto-optimise-store` is set to `true` (On Linux only)
```

Comment:
This should be turned off by default on linux, too since it slows down substitution significantly on big stores.

Reply:
Can you provide more information? How do big stores cause the substitution to slow down? I could imagine it if the store path is large, but not sure about stores.

Reply:
Yeah, typo, when downloading many or big things from the cache or remote builder while having auto-optimise-store turned on, it noticeable taker longer to complete.